src/config.c

```diff
@@ -852,6 +852,16 @@ void configSetCommand(redisClient *c) {
 
         if (flags == -1) goto badfmt;
         server.notify_keyspace_events = flags;
+    } config_set_special_field("rdbchecksum") {
+        if (server.server.loading == 1) {
```

Comment:
We could also just pull the value at the start and use that throughput the RDBLoad. I don't really like conditional failures.

Reply:
Agree